ACD-416 (#6)
Actindo-AG/Documentation: RetailSuiteAccounting/Operation/06_DeleteBookings.md

[!!User Interface bookings](../UserInterface/01a_Bookings.md)  
[!!User Interface delete](../UserInterface/01_Book.md#delete)  

# Delete bookings

> The *Accounting* module allows to delete bookings in the *BOOKINGS* tab, for example, if content errors are found or if the booking was created for test purposes.

Reviewer: ```suggestion
You can delete postings that are no longer needed, for example, if a posting is faulty.
```

> 
> It is possible to delete single bookings or multiple bookings at once. Bear in mind, however, that the bookings to be deleted must be unprocessed. For detailed information, see [Process bookings](./07_ProcessBookings.md). In fact, a booking that has already been processed cannot be deleted, as the [DELETE] button will change to inactive (grayed out) when clicking on a unprocessed booking.

Reviewer: ```suggestion
It is possible to delete a single posting or to delete multiple postings at once. Bear in mind, however, that the postings to be deleted must be unprocessed as postings that have already been processed cannot be deleted. For detailed information, see [Process postings](./07_ProcessBookings.md). 
```

## Delete a single booking

#### Prerequisites

> - The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
> - A valid fiscal year is created, see [Create a fiscal year](../Integration/04_ManageFiscalYear.md#create-a-fiscal-year).

Reviewer: ```suggestion
```

> - A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).

Author: ```suggestion
- A fiscal year has been selected, see [Select fiscal year](./01_SelectFiscalYear.md).
```

- A booking has been created, see [Create a manual booking](./04_CreateManualBooking.md).

#### Procedure

*Accounting > Select fiscal year > Book > Tab BOOKINGS*

> ![Delete booking](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteBooking.png "[Delete booking]")

Reviewer: Buchung in Bild noch nicht ausgewählt

> 1. Select the booking to be deleted from the list in the *BOOKINGS* tab.

Reviewer: ```suggestion
1. Select the posting to be deleted in the postings list.
```

> 1. Select the booking to be deleted from the list in the *BOOKINGS* tab.
> 
> 2. The existing booking details are displayed in the input line fields.

Reviewer: ```suggestion
1. Select the booking to be deleted  in the postings list.
  The posting details are displayed in the input line fields.
```

> 3. Click the [DELETE] button.  

Author: ```suggestion
2. Click the [DELETE] button.  
```

A confirmation window is displayed.

>   > [Warning] Be aware that the existing booking will be deleted permanently.

Reviewer: ```suggestion
  > [Warning] Be aware that the selected posting will be deleted permanently.
```

> 4. Click the [Ok] button to confirm the action.  

Reviewer: ```suggestion
4. Click the [Ok] button to confirm the action.  
```
```suggestion
4. Click the [OK] button to confirm the deletion.  
```

> A message above the input line confirms that the booking has been deleted. The deleted booking will no longer be displayed in the bookings list.

Reviewer: ```suggestion
The posting has been deleted. The notice "Posting deleted!" is displayed above the input line. The posting is removed from the postings list. 
```

  ![Booking deleted](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/BookingDeleted.png "[Booking deleted]")

## Delete multiple bookings

#### Prerequisites

> - The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
> - A valid fiscal year is created, see [Create a fiscal year](../Integration/04_ManageFiscalYear.md#create-a-fiscal-year).

Reviewer: ```suggestion
```

> - A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).

Author: ```suggestion
- A fiscal year has been selected, see [Select fiscal year](./01_SelectFiscalYear.md).
```

- Multiple bookings have been created, see [Create a manual booking](./04_CreateManualBooking.md).

#### Procedure

*Accounting > Select fiscal year > Book > Tab BOOKINGS > Context menu EDIT > Menu entry Delete*

> ![Delete multiple bookings](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteMultipleBookings.png "[Delete multiple bookings]")

Reviewer: Bild ohne Kontextmenü

> 

Reviewer: 1. Click the [EDIT] button in the header above the postings list.
   A context menu is displayed.

Bild Kontextmenü einfügen

> 1. Click the *Delete* menu entry in the *EDIT* context menu.   
> A window opens for you to select the journal and the sequential numbers of the bookings to be deleted.

Reviewer: ```suggestion
1. Click the *Delete...* menu entry in the context menu.     
  The *Delete bookings* window is displayed.
```

  ![Select multiple bookings](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteMultipleBookings01.png "[Select multiple bookings]")

2. Select the journal where the bookings to be deleted are recorded in the *Journal* drop-down list.

> 3. Enter the number range of the bookings to be deleted in the *Seq. numbers* field.

Reviewer: ```suggestion
4. Enter the number range of the postings to be deleted in the *Seq. numbers* field.
```

>   > [Warning] Be aware that the existing bookings will be deleted permanently.

Reviewer: ```suggestion
  > [Warning] Be aware that the selected postings will be deleted permanently.
```

>   ![Multiple bookings selected](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteMultipleBookings02.png "[Multiple bookings selected]")

Reviewer: Bild vor Warning

>   ![Multiple bookings selected](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteMultipleBookings02.png "[Multiple bookings selected]")

Reviewer: Bild kann eigentlich raus, zeigt keine große Veränderung...

4. Click the [DELETE] button.   

> The selected bookings are deleted and will no longer be displayed in the bookings list.

Reviewer: ```suggestion
The selected postings has been deleted. The postings are removed in the corresponding postings list.
```
Bei mir schließt das Fenster nicht automatisch - ist das normal? Dann schreiben, dass das Fenster geschlossen werden muss.

Author: Ich tippe eher auf ein Bug. Nach dem Klicken auf LÖSCHEN im Fenster, passiert nichts. Man muss F5 drücken, damit die Änderungen geziegt werden. Bug schon notiert in OneNote Dok.

> > [Info] It may be necessary to click on a different tab and switch back the *BOOKINGS* tab to display the changes in the bookings list.

Reviewer: Also das ist definitiv ein Bug. Müssen wir reporten. Kann ja nicht sein, dass man da erst so komisch rumwurschteln muss... 
Alternativ müssen wir eben sagen, dass man neu laden muss - hört sich besser an, als den Tab zu wechseln...

Author: S. mein Kommentar oben. :) Zwischenlösung: Schritt mit F5 zur Aktualisierung des Platforms, wie in Integration.

## Delete a single posting

Delete a single posting from the postings list.

> [Warning] Be aware that any deletion is permanent and cannot be undone.

#### Prerequisites

- A fiscal year has been selected, see [Select fiscal year](./01_SelectFiscalYear.md).
- At least one posting has been created, see [Create a manual posting](./04_CreateManualBooking.md).

#### Procedure

*Accounting > Post > Tab POSTINGS*

![Postings](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/Bookings.png "[Postings]")

1. Select the posting to be deleted in the postings list.  
  The posting details are displayed in the input line fields.

  > [Warning] Be aware that the selected posting will be deleted permanently.

2. Click the [DELETE] button.  
A confirmation window is displayed.

  ![Delete confirmation](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteConfirmation.png "[Delete confirmation]")

3. Click the [OK] button to confirm the deletion.  
The posting has been deleted. The notice "Posting deleted!" is displayed above the input line. The posting has been removed from the postings list.

## Delete multiple postings

> 

Reviewer 2: test

> Delete several postings simultaneously from the postings list.

Reviewer 2: ```suggestion
Delete several postings simultaneously from the postings list.
```
